### ISSUE TITLE
V4.2 will block query of Namespace/Class/Method references

### ISSUE THREAD

**Issue by lialan** [felixfbecker/php-language-server#240]
When using v4.2, when I try to query Namespace, Class or Method, the server seems to not able to return anything, resulting a freeze. Waited for a few mins and it is still like this. 

For example querying php-language-server with the following symbol info:

```
getting reference from symbol: SymbolInformation [
  name = "OtherClass"
  kind = Class
  location = Location [
    uri = "file:///pathtophplsp/php-language-server/fixtures/completion/fully_qualified_class.php"
    range = Range [
      start = Position [
        line = 8
        character = 0
      ]
      end = Position [
        line = 8
        character = 19
      ]
    ]
  ]
  containerName = "Whatever"
]
```

will result a freeze.

Step to reproduce:
1. download php language server (v4.2)
2. run "composer update"
3. client call initialization with rootpath to the target folder (in my case I used lsp4j)
4. query using an empty symbol string ""
5. use the returned symbol information to query references
6. see the communication freezes.

**Comment by felixfbecker** [comment]
Thanks for the report. What _methods_ are you calling exactly?

**Comment by lialan** [comment]
Hi @felixfbecker I used textDocument/references to retrieve the references.

The same method runs fine if I switch to use V3.1, but will not response on V4.2. Thanks!

**Comment by felixfbecker** [comment]
`textDocument/references` does not take a `SymbolInformation` as parameter though, so what parameters were you passing? Any STDERR output or an error response? Did you follow all installation instructions in the readme, like `composer run-script parse-stubs`? v3 -> v4 is a major version jump, so you have to expect breaking changes

**Comment by lialan** [comment]
Sorry should have explained in more detail. I constructed the ReferenceParams in the following way:

1. TextDocumentIdentifier.textDocument is set using the location from the SymbolInformation's location.uri
2. TTextDocumentIdentifier.position is set to use the SymbolInformation's range.start field
2. ReferenceContext is set to true

Prior to that I have parsed the stubs otherwise it will throw exceptions about cannot find stubs folder.

Does this explain it better?

**Comment by felixfbecker** [comment]
Yes, but I cannot reproduce. Do you have full logs of request/responses?
